mapping.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import collections
+
+df = pd.read_csv('RentData051022/medianAskingRent_All.csv')
```

Comment:
Let's also move this line into the function.  Also, lets drop any rows where `Borough` or `areaName` is null.

Reply:
why?

Reply:
Import should stay at the top of the file.  We'd just want to move `df = pd.read_csv('RentData051022/medianAskingRent_All.csv')` inside the function.  Its a best practice to not load dataframes in the main scope of a file.